app/src/main/res/drawable/button_new.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
rename this one to button_backgroung, if such element is exist rename that to old_button_background

Reply:
i'm rename on button_backgroung_new and all files which do'n have sufix new i'm delete on #127 ticket

Reply:
okay